plugins/org.jkiss.dbeaver.tasks.native/OSGI-INF/l10n/bundle_ro_RO.properties

```diff
@@ -0,0 +1 @@
+#Generated by ResourceBundle Editor (http://essiembre.github.io/eclipse-rbe/)
```

Comment:
No need in this empty file. Please, remove it.

Reply:
OK, but all the others are present here, so I thought it was necessary to create this empty file.

https://github.com/dbeaver/dbeaver/tree/devel/plugins/org.jkiss.dbeaver.tasks.native/OSGI-INF/l10n

Reply:
No. It was incorrect before. And we need to stop repeating this wrong behavior now.